service/src/main/java/bio/terra/pipelines/stairway/imputation/PrepareImputationInputsStep.java

```diff
@@ -0,0 +1,87 @@
+package bio.terra.pipelines.stairway.imputation;
+
+import bio.terra.pipelines.common.utils.FlightUtils;
+import bio.terra.pipelines.common.utils.PipelineInputTypesEnum;
+import bio.terra.pipelines.common.utils.PipelinesEnum;
+import bio.terra.pipelines.db.entities.PipelineInputDefinition;
+import bio.terra.pipelines.dependencies.stairway.JobMapKeys;
+import bio.terra.pipelines.service.PipelinesService;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import com.fasterxml.jackson.core.type.TypeReference;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.retry.RetryException;
+
+public class PrepareImputationInputsStep implements Step {
```

Comment:
made this imputation-specific

Reply:
can we add a java doc type explanation of this step